### PR TITLE
fix: Use concise text for CA popup

### PR DESCRIPTION
**Pull request by arajkumar**
- Update popup to show concise text (in sync with status bar text)
- Fix popup full stack report label to be consistent with status tool tip
- Don’t silence error popup

Screenshot 2020-12-23 at 12 49 26 PM

Also fixes https://issues.redhat.com/browse/APPAI-1679

### CODE REVIEW

**Review by dvandra — APPROVED** [review]
LGTM

**Review by dgpatelgit — APPROVED** [review]
lgtm

File: src/extension.ts [hunk 1]
```diff
@@ -123,11 +123,7 @@ export function activate(context: vscode.ExtensionContext) {
         lspClient.onNotification('caError', respData => {
           const notification = new CANotification(respData);
           caStatusBarProvider.setError();
-          if (canShowPopup(notification)) {
-            vscode.window.showErrorMessage(respData.data);
```
Comment (mathur07) [inline src/extension.ts:127]: Do we need to remove function declaration as-well?

**Review by mathur07 — APPROVED** [review]
lgtm

**Pull request merged**